src/main/java/com/googlecode/lanterna/gui2/LinearLayout.java

```diff
@@ -21,10 +21,7 @@
 import com.googlecode.lanterna.TerminalPosition;
 import com.googlecode.lanterna.TerminalSize;
 
-import java.util.ArrayList;
-import java.util.IdentityHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
I don't really like the "*" with imports...  they've bitten me in other projects already.

please rather list all the necessary classes...

Reply:
done